ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/AStarFootstepPlanner.java

```diff
@@ -302,13 +302,10 @@ private void reportStatus(FootstepPlannerRequest request, FootstepPlannerOutput
          outputToPack.getFootstepPlan().addFootstep(footstep);
       }
 
-      if (!request.getAssumeFlatGround())
-      {
-         swingPlanningModule.computeSwingWaypoints(request.getHeightMapData(),
-                                                   outputToPack.getFootstepPlan(),
-                                                   request.getStartFootPoses(),
-                                                   request.getSwingPlannerType());
-      }
+      swingPlanningModule.computeSwingWaypoints(request.getHeightMapData(),
+                                                outputToPack.getFootstepPlan(),
+                                                request.getStartFootPoses(),
+                                                request.getSwingPlannerType());
```

Comment:
why did this get pulled out of its `if` case?

Reply:
Because if the height map is null it returns from that method, and its not a large computation so we can always check to see if we have a height map, and compute swing waypoints if needed